data/src/commonMain/kotlin/com/wire/kalium/logic/data/message/AssetContent.kt

```diff
@@ -18,8 +18,6 @@
 
 package com.wire.kalium.logic.data.message
 
-import com.wire.kalium.logic.sync.receiver.conversation.message.hasValidRemoteData
```

Comment:
This was just in a very weird package.

Doesn't make sense to import things from `conversation / receiver` in order to check if asset content has valid remote data